src/main/java/uk/gov/companieshouse/api/accounts/model/rest/smallfull/notes/stocks/Stocks.java

```diff
@@ -1,13 +1,15 @@
-package uk.gov.companieshouse.api.accounts.model.rest.notes.stocks;
+package uk.gov.companieshouse.api.accounts.model.rest.smallfull.notes.stocks;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import uk.gov.companieshouse.api.accounts.model.rest.RestObject;
+import uk.gov.companieshouse.api.accounts.model.rest.Note;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.stocks.CurrentPeriod;
```

Comment:
Shall we move these classes into the same package as `Stocks` no exists, and bin the `uk.gov.companieshouse.api.accounts.model.rest.notes.stocks` package?